src/browsers/browser-creator.ts

```diff
@@ -52,7 +53,8 @@ export default abstract class BrowserCreator {
     await browser.setTimeout({ implicit: options.implicitTimeout, script: options.scriptTimeout });
 
     if (!browser.isMobile) {
-      await browser.$('body').then(body => body.moveTo({ xOffset: 0, yOffset: 0 }));
+      const body = await browser.$('//body');
+      body.moveTo({ xOffset: 0, yOffset: 0 });
```

Comment:
What is the return of `moveTo`? Is it a promise?

Reply:
ah good catch! webdriverio 9 does a lot more promise chaining magic under the hood, so it _might_ be that it handles this anyway, but makes sense to make it more explicit